### ISSUE TITLE
Module code is not efficient

### ISSUE THREAD

**Issue by virtuald** [robotpy/robotpy-websim#23]
My CPU runs pretty high when running simulation. There's a number of issues that are easily fixable:

* Shouldn't be selecting/finding HTML DOM elements each iteration. Do it once, store them, and only use when necessary
* Probably shouldn't be updating DOM elements unless they change
* Shouldn't be adding/removing classes each loop iteration -- only do it when needed

You'll note there's a theme here...

**Comment by virtuald** [comment]
Yeah, we need to fix this. All of my students batteries are getting killed when we run the websim.

**Comment by amorygalili** [comment]
The DOM elements are cached and the DOM elements don't change unless new data is sent from the server. Not completely optimized, but it should be better now. Let me know if it's still killing your students' batteries.
